### PR TITLE
Fix kwargs in clip and logical API

### PR DESCRIPTION
**Pull request by velconia**

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -29,110 +29,29 @@
...
-    'sequence_concat',
-    'scale',
-    'elementwise_add',
-    'elementwise_div',
-    'elementwise_sub',
-    'elementwise_mul',
-    'elementwise_max',
-    'elementwise_min',
-    'elementwise_pow',
+    'fc', 'embedding', 'dynamic_lstm', 'dynamic_lstmp', 'dynamic_gru',
```
Comment (panyx0718) [inline python/paddle/fluid/layers/nn.py]: Would be better to keep the original format.

**Review by panyx0718 — DISMISSED** [review]

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (velconia) [reply python/paddle/fluid/layers/nn.py]: The format was changed by pylint

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -29,110 +29,29 @@
...
+    'lod_reset', 'lrn', 'pad', 'pad_constant_like', 'label_smooth', 'roi_pool',
+    'dice_loss', 'image_resize', 'image_resize_short', 'resize_bilinear',
+    'gather', 'scatter', 'sequence_scatter', 'random_crop', 'mean_iou', 'relu',
+    'log', 'crop', 'rank_loss', 'elu', 'relu6', 'pow', 'stanh', 'hard_sigmoid',
+    'swish', 'prelu', 'brelu', 'leaky_relu', 'soft_relu', 'flatten',
+    'sequence_mask', 'stack', 'pad2d', 'unstack', 'sequence_enumerate',
+    'expand', 'sequence_concat', 'scale', 'elementwise_add', 'elementwise_div',
+    'elementwise_sub', 'elementwise_mul', 'elementwise_max', 'elementwise_min',
+    'elementwise_pow', 'logical_and', 'logical_or', 'logical_xor',
+    'logical_not', 'clip', 'clip_by_norm'
```
Comment (qingqing01) [inline python/paddle/fluid/layers/nn.py]: 似乎最后加逗号，不会被格式化。

**Review by qingqing01 — DISMISSED** [review]

**Review by reyoung — DISMISSED** [review]

File: paddle/fluid/API.spec [hunk 3]
```diff
@@ -160,6 +160,12 @@ paddle.fluid.layers.gaussian_random_batch_size_like ArgSpec(args=['input', 'shap
 paddle.fluid.layers.sum ArgSpec(args=['x', 'use_mkldnn'], varargs=None, keywords=None, defaults=(False,))
 paddle.fluid.layers.slice ArgSpec(args=['input', 'axes', 'starts', 'ends'], varargs=None, keywords=None, defaults=None)
 paddle.fluid.layers.shape ArgSpec(args=['input'], varargs=None, keywords=None, defaults=None)
+paddle.fluid.layers.logical_and ArgSpec(args=['x', 'y', 'out', 'name'], varargs=None, keywords=None, defaults=(None, None))
```
Comment (panyx0718) [inline paddle/fluid/API.spec:163]: Can we hide out?

**Review by panyx0718 — APPROVED** [review]

File: paddle/fluid/API.spec (hunk 3 above)
Comment (velconia) [reply paddle/fluid/API.spec:163]: It's used in models for while op

Comment (velconia) [reply paddle/fluid/API.spec:163]: Could you please merge this?

**Pull request merged**